internal/stack/_static/Dockerfile.logstash

```
@@ -0,0 +1,10 @@
+ARG IMAGE
+FROM ${IMAGE}
+
+RUN bin/logstash-plugin install logstash-filter-elastic_integration
```

This will always pull new version of the plugin, the expectation to use as it is if plugin is bundled with current Logstash version. Would be problem with other plugins but this one is tightly coupled with ES.

```suggestion
RUN if [ ! $(bin/logstash-plugin list) == *logstash-filter-elastic_integration* ]; then \
        bin/logstash-plugin install logstash-filter-elastic_integration; \
    fi
```

FYI: couldn't test if this works...

---

Change applied, thanks.